src/components/Collection/Sidebar.css

```diff
@@ -39,6 +39,7 @@
     position: absolute;
     left: 6px;
     z-index: 2;
+    top: 9px; /* issue #907 */
```

Comment:
For me, this was `9.4px`, can you adjust it a bit?

Reply:
I'd rather avoid partial pixels, and I also want to avoid locking down all browsers to this solution if possible. Taking a quick look locally.